tools/ci_connector_ops/ci_connector_ops/pipelines/bases.py

```diff
@@ -287,6 +344,7 @@ class Report:
     created_at: datetime = field(default_factory=datetime.utcnow)
     name: str = "REPORT"
     filename: str = "output"
+    extra_files_to_upload: Optional[List[Tuple[Path, str]]] = field(default_factory=list)
```

Comment:
Question: Where is this set?

Reply:
It's not used anywhere ATM but I figured it might be helpful. I originally used it to upload to dagger logs to GCS, but the final approach I took is better as it does not rely on dagger engine running to upload logs...